magefile.go

```diff
@@ -62,6 +62,19 @@ func Test() error {
 	return nil
 }
 
+// IntegrationTest runs the integration test functions
+func IntegrationTest() error {
+	if err := mage.EnsureGitConfig(); err != nil {
+		return err
+	}
```

Comment:
This is technically not needed for this PR, but it'll be needed for git integration tests.